modulefiles/build.wcoss2.intel.lua

```diff
@@ -51,8 +51,7 @@ load(pathJoin("sigio", sigio_ver))
 sp_ver=os.getenv("sp_ver") or "2.3.3"
 load(pathJoin("sp", sp_ver))
 
-ip_ver=os.getenv("ip_ver") or "3.3.3"
-load(pathJoin("ip", ip_ver))
+-- setenv("ip_DIR","/lfs/h2/emc/global/noscrub/George.Gayno/ip.git/lib64/cmake/ip")
 
```

Comment:
@GeorgeGayno-NOAA, so the new version of the IP library was built in your own user space?  Is there a general user space on WCOSS2 where it could go instead?

Reply:
That was a comment that should have been removed. 

I also removed the specification of the MODULEPATH below that was required to find the new library.  I see that is no longer needed. (See a682ce8). These machines are always in a state of flux.

Thanks for your careful review.

Reply:
Oh, yes, sorry.  I missed that this was a comment!   Thanks for making these changes.